scripts/extent/shpToBbox.R

```diff
@@ -0,0 +1,26 @@
+
+## Install required packages
+packages <- c("rjson", "dplyr")
+new.packages <- packages[!(packages %in% installed.packages()[,"Package"])]
+if(length(new.packages)) install.packages(new.packages)
+
+## Load required packages
+
+library("rjson")
+library("raster")
+library("dplyr")
+## Load functions
+source(paste(Sys.getenv("SCRIPT_LOCATION"), "SDM/sdmUtils.R", sep = "/"))
+
+
+input <- fromJSON(file=file.path(outputFolder, "input.json"))
+print("Inputs: ")
+print(input)
+
+shp <- sf::st_read(input$extent)
+bbox <- shp_to_bbox(shp, proj_to = "EPSG:4326")
```

Comment:
I am using ./SDM/extentToBbox.yml to convert spatial object to a bbox. So, I am not sure who is using this function.  May be Guillaume?

Reply:
Certainly not, since the file was added in this branch and is not present in the others.
Ok, so you are always using extentToBbox, then we can remove this one and associated examples...